### ISSUE TITLE
bulk upload - complete event error

### ISSUE THREAD

**Issue by nbaertlein** [USGS-WiM/whispersservices_django#329]
Can't get a "complete" event to load. Error about 'event, diagnosis' being unique even though there is only 1 record.  Trial csv attached (zipped).  Other business rules constraints appear to be valid.
[test_upload15_fail_error_uniqueeventset.zip](https://github.com/USGS-WiM/whispers/files/4042823/test_upload15_fail_error_uniqueeventset.zip)

![image](https://user-images.githubusercontent.com/38669574/72110137-180f6c00-32fd-11ea-9881-e58c57740bd2.png)

**Comment by BlakeDraper** [comment]
@aaronstephenson I am not seeing what it is about Neil's upload that is triggering an error. I have included the JSON submission (as generated by the client parsing logic) to stick directly into Insomnia:


```
{
  "event_reference": "BA2",
  "event_type": 1,
  "public": true,
  "complete": true,
  "new_event_diagnoses": [
    {
      "diagnosis": 469
    }
  ],
  "new_event_locations": [
    {
      "name": "Marco Island",
      "start_date": "2019-12-29",
      "end_date": "2019-12-31",
      "country": 30,
      "administrative_level_one": 10,
      "administrative_level_two": 328,
      "latitude": 25.9325,
      "longitude": -81.682,
      "land_ownership": 8,
      "history": "This event location was added using the bulk upload process.",
      "new_location_species": [
        {
          "species": 297,
          "population_count": null,
          "sick_count": null,
          "dead_count": 4,
          "sick_count_estimated": null,
          "dead_count_estimated": null,
          "captive": false,
          "new_species_diagnoses": [
            {
              "diagnosis": 469,
              "cause": 1,
              "basis": 1,
              "suspect": true,
              "tested_count": 0,
              "diagnosis_count": 0,
              "new_species_diagnosis_organizations": []
            }
          ]
        }
      ]
    }
  ]
}

I get the same unique set message, but I am not following what it means when it says the event and diagnosis must make a unique set, since it can't mean that two events can't have the same diagnosis. Since this is the creation of a new event based on the provided data, what is the conflict?

 My thought was Neil was posting the same data over and over and that there was a check for that but I made tweaks to this submission and got the same outcome. Is there something malformed in the submission maybe?
```

**Comment by aaronstephenson** [comment]
At first glance, knowing that Diagnosis #469 is `Undetermined`, I'm guessing that the submitted value is conflicting with the automatically-created EventDiagnosis. Essentially, the Undetermined EventDiagnosis already exists (when the Event is created, before the submitted EventDiagnosis is even assessed) and the submitted diagnosis is superfluous. I thought we had taken care of this months ago, but maybe I missed something. I'll take a closer look.

**Comment by aaronstephenson** [comment]
Fixed in 1.4.5 bb9c270
